exercises/alphanumeric-pattern-generator/alphanumeric_pattern_generator_test.rb

```diff
@@ -0,0 +1,156 @@
+require 'minitest/autorun'
+require_relative 'alphanumeric_pattern_generator'
+
+class PatternGeneratorTest < Minitest::Test
+  def test_it_can_verify_a_single_alphabetic_character
+    pattern = '.'
+    pg = PatternGenerator.new(pattern)
+
+    assert_equal true, pg.verify('F')
+    assert_equal false, pg.verify('0')
+  end
+
+  def test_it_verifies_upper_and_lowercase_characters
+    pattern = '.'
+    pg = PatternGenerator.new(pattern)
+
+    assert_equal true, pg.verify('r')
+    assert_equal true, pg.verify('R')
+  end
+
+  def test_it_generates_a_single_character_pattern
+    pattern = '.'
+    pg = PatternGenerator.new(pattern)
+
+    assert /[A-Z]/.match(pg.generate)
+    refute /[0-9]/.match(pg.generate)
```

Comment:
Try and be consistent about usage of `assert/refute` and `assert_equal true/assert_equal false`
I'd prefer using `assert` and `refute` everywhere.

Reply:
The consistency was broken due to the truthy/ falsey nature of `match`. I try to stay away from `assert` and `refute` if I am specifically looking for the values `true` and `false`, hence the reason I am mostly using `assert_equal` throughout the test suite. Testing regex with match returns either an integer or nil which is why I changed my patter to `assert` and `refute`.

Reply:
Fair enough. I will change them to `assert_equal 0, /[A-Z]/.match(pg.generate)`. Thank you.